### PR TITLE
Added host_conv_wrw for verification

### PR DESCRIPTION
**Pull request by zjing14**
- Added host_direct_convolution_backward_weights function

### CODE REVIEW

File: host/host_tensor/include/host_conv_wrw.hpp [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          typename TOut,
+          typename TWei,
+          typename ConvStrides,
+          typename ConvDilations,
+          typename InLeftPads,
+          typename InRightPads>
+void host_direct_convolution_backward_weights(
+    const Tensor<TIn>& in,
+    const Tensor<TOut>& out,
+    Tensor<TWei>& wei,
```
Comment (ltqin) [inline host/host_tensor/include/host_conv_wrw.hpp:14]: in wei out or out in wei?

**Comment by ltqin** [comment]
@zjing14  I didn't find out issue on code, but the result is not right.
branch: backward_weight_v4r4r2_xdlops
command: host/driver_offline/conv_wrw_driver_offline  0 0 1  1  0  1  128  256  128 3 3  14  14  1  1  1  1  0  0  0  0
device result: 18432
host    result: 4608

**Comment by zjing14** [comment]
> @zjing14 I didn't find out issue on code, but the result is not right.
> branch: backward_weight_v4r4r2_xdlops
> command: host/driver_offline/conv_wrw_driver_offline 0 0 1 1 0 1 128 256 128 3 3 14 14 1 1 1 1 0 0 0 0
> in: dim 4, lengths {128, 128, 14, 14}, strides {25088, 196, 14, 1}
> wei: dim 4, lengths {256, 128, 3, 3}, strides {1152, 9, 3, 1}
> out: dim 4, lengths {128, 256, 12, 12}, strides {36864, 144, 12, 1}
> 
> device result: 18432
> host result: 4608

There is a bug in the function. FIxed. Please pull and check.

File: host/host_tensor/include/host_conv_wrw.hpp (hunk 1 above)
Comment (zjing14) [reply host/host_tensor/include/host_conv_wrw.hpp:14]: (in, out, wei)

**Review by asroy — APPROVED** [review]

**Comment by asroy** [comment]
@ltqin Please check if GPU and CPU result match

**Review by ltqin — APPROVED** [review]

**Pull request merged**